PythagoreanTheorem.java

```diff
@@ -1,15 +1,61 @@
-/*
- * File: PythagoreanTheorem.java
- * Name: 
- * Section Leader: 
- * -----------------------------
- * This file is the starter file for the PythagoreanTheorem problem.
- */
-
-import acm.program.*;
-
-public class PythagoreanTheorem extends ConsoleProgram {
-	public void run() {
-		/* You fill this in */
-	}
-}
+/*
+ * File: PythagoreanTheorem.java
+ * Name: 
+ * Section Leader: 
+ * -----------------------------
+ * This file is the starter file for the PythagoreanTheorem problem.
+ */
+
+import acm.program.*;
+import java.util.Scanner;
+
+
+public class PythagoreanTheorem extends ConsoleProgram {
+	
+		 public static void main(String[] args){
+				
+				double legA, legB;
+			
+				
+				//Declare a String object to store the uer's name, 
+				
+				
+				//Create a Scanner object for reading in the user's input
+				Scanner input= new Scanner(System.in);
+
+				
+				//Prompt the user for the lengths of the legs, 
+				//and store those two pieces of information in legA and legB.
+				System.out.println("Please enter the length of the "

```

Comment:
You should use ConsoleProgram printing method so this should be
println("Please enter the length of the...") instead of
System.out.println("Please enter the length of the...")